tests/autopilot-capacity/aut_test.go

```diff
@@ -41,63 +102,118 @@ var _ = Describe(fmt.Sprintf("{%sBasic}", testNameSuite), func() {
 		var err error
 		testName := strings.ToLower(fmt.Sprintf("%sBasic", testNameSuite))
 
-		apParameters := &scheduler.AutopilotParameters{
-			Enabled: true,
-			Name:    testName,
-			AutopilotRuleParameters: scheduler.AutopilotRuleParameters{
-				ActionsCoolDownPeriod: 60,
-				PVCWorkloadSize:       10737418240, //10Gb
-				PVCPercentageUsage:    50,
-				PVCPercentageScale:    50,
-			},
-		}
-
-		Step("schedule applications", func() {
-			for i := 0; i < Inst().ScaleFactor; i++ {
-				taskName := fmt.Sprintf("%s-%v", fmt.Sprintf("%s-%d", testName, i), Inst().InstanceID)
-				context, err := Inst().S.Schedule(taskName, scheduler.ScheduleOptions{
-					AppKeys:             Inst().AppList,
-					StorageProvisioner:  Inst().Provisioner,
-					AutopilotParameters: apParameters,
-				})
-				Expect(err).NotTo(HaveOccurred())
-				Expect(context).NotTo(BeEmpty())
-				contexts = append(contexts, context...)
+		for _, apRule := range AutopilotruleBasicTestCases {
+			apParameters := &scheduler.AutopilotParameters{
+				Enabled:                 true,
+				Name:                    testName,
+				AutopilotRuleParameters: apRule,
 			}
-		})
 
-		Step("wait until workload completes on volume", func() {
-			for _, ctx := range contexts {
-				err = Inst().S.WaitForRunning(ctx, workloadTimeout, retryInterval)
-				Expect(err).NotTo(HaveOccurred())
-			}
-		})
-
-		Step("validating volumes and verifying size of volumes", func() {
-			for _, ctx := range contexts {
-				err = Inst().S.InspectVolumes(ctx, timeout, retryInterval)
-				Expect(err).NotTo(HaveOccurred())
-			}
-		})
+			Step("schedule applications", func() {
+				for i := 0; i < Inst().ScaleFactor; i++ {
+					taskName := fmt.Sprintf("%s-%v", fmt.Sprintf("%s-%d", testName, i), Inst().InstanceID)
+					context, err := Inst().S.Schedule(taskName, scheduler.ScheduleOptions{
+						AppKeys:             Inst().AppList,
+						StorageProvisioner:  Inst().Provisioner,
+						AutopilotParameters: apParameters,
+					})
+					Expect(err).NotTo(HaveOccurred())
+					Expect(context).NotTo(BeEmpty())
+					contexts = append(contexts, context...)
+				}
+			})
+
+			Step("wait until workload completes on volume", func() {
+				for _, ctx := range contexts {
+					err = Inst().S.WaitForRunning(ctx, workloadTimeout, retryInterval)
+					Expect(err).NotTo(HaveOccurred())
+				}
+			})
+
+			Step("validating volumes and verifying size of volumes", func() {
+				for _, ctx := range contexts {
+					err = Inst().S.InspectVolumes(ctx, timeout, retryInterval)
+					Expect(err).NotTo(HaveOccurred())
+				}
+			})
+
+			Step("wait for unscheduled resize of volume (10min)", func() {
+				time.Sleep(10 * time.Minute)
```

Comment:
move `10 * time.Minute` as a const at top of the file. Do this for all constants moving forward